components/kyma-environment-broker/internal/process/upgrade_kyma/ems_prov_int_test.go

```diff
@@ -105,85 +105,3 @@ func TestEmsUpgradeProvisioningSteps(t *testing.T) {
 	fmt.Printf("\nexport INSTANCE_ID=%s\nexport BINDING_ID=%s\n", operation.Ems.Instance.InstanceID, operation.Ems.BindingID)
 	fmt.Printf("\nexport OVERRIDES=%#v\n", overridesOut)
 }
-
-func newInputCreator() *simpleInputCreator {
```

Comment:
These were outdated and masked by a build tag in this file . I moved them to the `automock` package since and added the messing functions.